test/controllers/graphql_controller_10_test.rb

```diff
@@ -769,13 +769,17 @@ def setup
     create_team_user user: u, team: t, role: 'editor'
     pm = create_project_media team: t
     tr = create_tipline_request team_id: t.id, associated: pm, language: 'en', smooch_data: { authorId: '123', language: 'en', received: Time.now.to_f }
+    assert_equal 0, tr.reload.first_manual_response_at
+    assert_equal 0, tr.reload.last_manual_response_at
     authenticate_with_user(u)
 
     query = "mutation { sendTiplineMessage(input: { clientMutationId: \"1\", message: \"Hello\", inReplyToId: #{tr.id} }) { success } }"
     post :create, params: { query: query, team: t.slug }
 
     assert_response :success
     assert JSON.parse(@response.body)['data']['sendTiplineMessage']['success']
+    assert tr.reload.first_manual_response_at > 0
+    assert tr.reload.last_manual_response_at > 0
   end
```

Comment:
I recommend adding another test in which first_manual_response_at does not update, while last_manual_response_at should be updated with a new value.

Reply:
Done!